### PR TITLE
Add `.SelectControl` styles

### PR DESCRIPTION
**Pull request by ohskylab**
TP: https://rockabox.tpondemand.com/entity/9179

Temporary, to tide us over until a custom typeahead is built.

Uses the exciting, new(ish) `appearance` property. This should work in Chrome, Firefox,
other Webkit but may fail horribly in IE. C'est la vie.

### CODE REVIEW

File: src/rb-select-control/demo/demo.tpl.html [hunk 1]
```diff
@@ -1,10 +1,94 @@
...
+                </label>
+                <select class="SelectControl-field is-invalid" id="SelectDemoInvalid">
+                    <option value="">Car</option>
+                    <option value="0" label="Volvo">Volvo</option>
+                    <option value="1" label="Audi">Audi</option>
+                    <option value="2" label="BMW">BMW</option>
+                    <option value="3" label="Mercedes">Mercedes</option>
+                </select>
+                <div class="SelectControl-message is-invalid">
+                    i.e. Your favourite make
```
Comment (chrisvxd) [inline src/rb-select-control/demo/demo.tpl.html]: This is a weird example `is-invalid` message

**Comment by chrisvxd** [comment]
Can't help but feel like the dropdown arrow should react when clicked

File: src/rb-select-control/rb-select-control.css [hunk 2]
```diff
@@ -5,12 +5,117 @@
...
+}
+
+/**
+ * Default
+ */
+
+.SelectControl-field {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none; /* 1 */
```
Comment (chrisvxd) [inline src/rb-select-control/rb-select-control.css]: Does post-css not handle `-moz` and `-webkit` properties

**Comment by ohskylab** [comment]
Need a fix for FF, dropping this back to DNM.

File: src/rb-select-control/demo/demo.tpl.html (hunk 1 above)
Comment (ohskylab) [inline src/rb-select-control/demo/demo.tpl.html]: Agree, fixed up.

**Comment by ohskylab** [comment]
> Can't help but feel like the dropdown arrow should react when clicked.

@chrisvxd Yes, but let's stick to the component PSDs and WoT. We'll most likely drop this component out and replace with a custom typeahead in the not-to-distant future anyway.

File: src/rb-select-control/rb-select-control.css (hunk 2 above)
Comment (ohskylab) [inline src/rb-select-control/rb-select-control.css]: See [referenced comment](https://github.com/rockabox/rbx_ui_components/pull/146/files#diff-cb970ae2b88c8c9485c02e8cf985a31dR28), which links to explanation at http://git.io/vTwoV

**Comment by ohskylab** [comment]
Mozilla fix in.

File: src/rb-select-control/rb-select-control.css (hunk 2 above)
Comment (chrisvxd) [inline src/rb-select-control/rb-select-control.css]: @ohskylab my bad.

**Pull request merged**